cmds/exp/gosh/gosh.go

```diff
@@ -162,13 +112,13 @@ func (s shell) runInteractiveTabCompletion(r *interp.Runner, stdout io.Writer) e
 				return true
 			}
 
-			runErr = r.Run(context.Background(), stmt)
```

Comment:
Should we maybe print out any occurence of runErr instead of ignoring it here?

Reply:
Printing this error would result in error code 127 after EVERY use of non existing commands.

Reply:
Oh yes, you're right!